osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -157,15 +158,18 @@ protected override void OnHoverLost(HoverLostEvent e)
 
         private void updateDisplay()
         {
-            if (isVoted.Value)
+            if (Action != null)
```

Comment:
A nitpick, but I'd invert this (check for null and immediately return if null, like a guard). Reduces nesting and makes it a bit easier to read for me.